Tests/NostrSDKTests/NIP44v2EncryptingTests.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  NIP44v2EncryptingTests.swift
+//  
+//
+//  Created by Terry Yiu on 3/17/24.
+//
+
+import XCTest
+import CryptoKit
+@testable import NostrSDK
+
+final class NIP44v2EncryptingTests: XCTestCase, NIP44v2Encrypting {
+
+    private static let vectors: NIP44Vectors? = try? FixtureLoader.shared.decodeFixture(filename: "nip44.vectors")
```

Comment:
This can be simplified:

```suggestion
    private lazy var vectors: NIP44Vectors = try! decodeFixture(filename: "nip44.vectors")  // swiftlint:disable:this force_try
```

Now you can...
1. Remove the `setUpWithError` function entirely.
2. Replace all `Self.vectors?.` with `vectors.`.
3. Conform this test class to `FixtureLoading` and remove the `FixtureLoader` singleton.